cmd/airplane/tasks/initcmd/init.go

```diff
@@ -115,7 +117,8 @@ func Run(ctx context.Context, cfg config) error {
 	if cfg.codeOnly && cfg.from == "" {
 		return errors.New("Required flag(s) \"from\" not set")
 	}
-	if cfg.codeOnly && cfg.inline {
+	isInline := cfg.inline || cfg.workflow
```

Comment:
This might be less error-prone. If we do this, can we remove all the `isInline` checks below and consult `cfg.inline` instead?

```suggestion
  cfg.inline = cfg.inline || cfg.workflow
```